CHANGES.md

```diff
@@ -504,20 +504,24 @@ OpenSSL 3.0
 
  * All of the low level DSA functions have been deprecated including:
 
+   DSA_SIG_new, DSA_SIG_free, DSA_SIG_get0, DSA_SIG_set0, DSA_new, DSA_free,
```

Comment:
DSA_SIG related functions are not deprecated.

Reply:
Fixed